### ISSUE TITLE
Shape properties are not stored

### ISSUE THREAD

**Issue by GoogleCodeExporter** [luisvt/oryx-editor#98]
```
All shape properties where the properties'id contain upper case letters,
are not stored, because RDF predicates must be upper case.


```

Original issue reported on code.google.com by `NicoPete...@gmail.com` on 6 May 2008 at 2:20

**Comment by GoogleCodeExporter** [comment]
```
fixed in issue #246
```

Original comment by `NicoPete...@gmail.com` on 6 May 2008 at 7:05
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `NicoPete...@gmail.com` on 23 May 2008 at 3:26
- Changed state: **Verified**
